app/styles/common.less

```diff
@@ -30,7 +42,7 @@ h1 {
   position: absolute;
   top: @status-bar-height;
   left: @side-nav-width;
-  width:  calc(~"100vw -" @side-nav-width);
+  width:  calc(~"100vw -" @side-nav-width + 15px);
```

Comment:
All views seem to look fine. The problem this was addressing is that when the page has a scroll bar the page is too big for the window, and results in a really ugly horizontal scrolling situation. Was unsure how to get around it other than by doing this.